mkosi/__init__.py

```diff
@@ -2361,6 +2326,27 @@ def load_credentials(args: argparse.Namespace) -> dict[str, str]:
     return creds
 
 
+def load_kernel_command_line_extra(args: argparse.Namespace) -> list[str]:
+    cmdline = []
+
+    for s in args.kernel_command_line_extra:
+        key, sep, value = s.partition("=")
+        if " " in value:
+            value = f'"{value}"'
```

Comment:
Couldn't we just indiscriminately call `shlex.quote` here instead?

Reply:
I'm not sure, I wouldn't risk it